### PR TITLE
travis config

### PR DESCRIPTION
**Pull request by Grummfy**

### CODE REVIEW

**Comment by Grummfy** [comment]
@jubianchi , @mikaelrandy or @Hywan I have remove hhvm if it's ok for you.
I temporary make a remove because of some issue with the cache of atoum in travis.
Otherwise I fix the dependency that make the build failed.

**Comment by Grummfy** [comment]
for appveyor don't know how to fix it

**Comment by Grummfy** [comment]
Appvoyer still break but not atoum

File: .travis.yml [hunk 1]
```diff
@@ -28,7 +24,8 @@ env:
         - secure: "vJ1TnFhU9muqLwkiBd13lFqdPJIrwKJ5OJjR3/jQROWQsRO4NPd/I7OfdDKk\noJGYbgjGMw7bmBYI0xjhYfZAnrClwIBvn1Qx5x5UDGxTHC8ZP6xs49AGa37I\n7BJlWC7+nRwPmmLhoLsbut/O7Kkn8IkdJIeChHbdn8RHQIJTJ4Y="
 sudo: false
 before_script:
-  - composer require --ignore-platform-reqs 'atoum/reports-extension:^2.0.1'
+  - composer remove 'atoum/reports-extension'
```
Comment (mikaelrandy) [inline .travis.yml]: Why do you need to remove this ?

Comment (Grummfy) [reply .travis.yml]: because of cache of travis (could be removed after the merge)

**Pull request merged**